packages/nextjs/nextjs.typedoc.json

```diff
@@ -26,12 +26,12 @@
       "@nhost/hasura-storage-js": [
         "../hasura-storage-js/src/index.ts"
       ],
+      "@nhost/nhost-js": [
```

Comment:
why was this change necessary?

Reply:
for the same reason we discussed about: typedoc fails with a heap out of memory error. When setting the path to the transpilted dts, typedoc succeeds

Reply:
Would this change solve the issue for `nhost-js`'s docgen as well? Now all of the references to docgen were removed from `nhost-js`, but I'd assume that pointing everything to transpiled code would solve the issue there too. Have you tried doing that?

Reply:
It passes the build, but I haven't checked all the links between markdown files. That's why I avoided setting it that way everywhere. For instance, if an interface is defined in `@nhost/hasura-auth-js` but the `@nhost/hasura-auth-js` path resolves to the transpiled dts, then the markdown file that describes this interface won't be rendered nicely.